src/components/Tooltips/CanvasHoverTooltip.tsx

```diff
@@ -3,7 +3,7 @@ import React, { PropsWithChildren, ReactElement, useCallback } from "react";
 import styled from "styled-components";
 import { useShallow } from "zustand/shallow";
 
-import { AnnotationSelectionMode, VECTOR_KEY_MOTION_DELTA, VectorTooltipMode } from "../../colorizer";
+import { AnnotationSelectionMode, PixelIdInfo, VECTOR_KEY_MOTION_DELTA, VectorTooltipMode } from "../../colorizer";
```

Comment:
I recommend turning off whitespace when reviewing this file, since I had to add a few conditionals that caused a lot of indenting.